meinberlin/static/embed.js

```diff
@@ -138,6 +138,7 @@ $(document).ready(function () {
     $error.text(text)
     $error.append($close)
     $error.prependTo($('#embed-status'))
+    $close.attr('title', django.gettext('Close'))
```

Comment:
Both `title` and `aria-label` have pros and cons. I would like to discuss this separately.